### PR TITLE
refactor: updated docs for oracle-node `v1.1.0` release

### PR DESCRIPTION
**Pull request by Yashk767**

### CODE REVIEW

**Comment by Yashk767** [comment]
Have covered all the tasks mentioned here - https://linear.app/interstellar-research/issue/RAZ-420
cc @yohanelly95 @rajkharvar

**Review by rajkharvar — APPROVED** [review]

File: docs/stake/mainnet.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ To update the razor-go node version
 2. Check your container is running via `docker ps`, you should get an output like:
     ```
     CONTAINER ID   IMAGE                                  COMMAND     CREATED         STATUS         PORTS     NAMES
-    5f0b7d99a71b   razornetwork/razor-go:v1.0.4           "/bin/sh"   3 weeks ago     Up 3 weeks               razor-go
+    5f0b7d99a71b   razornetwork/razor-go:v1.0.6           "/bin/sh"   3 weeks ago     Up 3 weeks               razor-go
```
Comment (yohanelly95) [inline docs/stake/mainnet.md:217]: @Yashk767 does this need to be v1.1.0 ?

Comment (yohanelly95) [reply docs/stake/mainnet.md:217]: never mind makes sense

Comment (Yashk767) [reply docs/stake/mainnet.md:217]: No, the `docker ps` will show the current docker image which is currently running which is expected.

**Review by yohanelly95 — APPROVED** [review]

**Pull request merged**